bin/runtime/src/lib.rs

```diff
@@ -1095,6 +1070,10 @@ impl_runtime_apis! {
         ) -> pallet_transaction_payment_rpc_runtime_api::RuntimeDispatchInfo<Balance> {
             TransactionPayment::query_info(uxt, len)
         }
+        // TODO check if it's needed, it exists in polkadot-sdk 1.4.0 fake_runtime.rs
```

Comment:
should we have a Jira for that? or let know @lesniak43 who is working on https://github.com/Cardinal-Cryptography/aleph-node/pull/1650?

Reply:
I've created a JIRA